mod-kb-ebsco/mod-kb-ebsco-java.postman_collection.json

```diff
@@ -7208,6 +7192,46 @@
 					],
 					"_postman_isSubFolder": true
 				}
+			],
+			"event": [
+				{
+					"listen": "prerequest",
+					"script": {
+						"id": "88241cbf-559b-4e80-a29d-281c6147fb8f",
+						"type": "text/javascript",
+						"exec": [
+							""
+						]
+					}
+				},
+				{
+					"listen": "test",
+					"script": {
+						"id": "ddd78cdc-06a1-4430-bd18-8fd2ca230fd0",
+						"type": "text/javascript",
+						"exec": [
+							"tv4.addSchema(\"schema_providers.json\", JSON.parse(pm.variables.get(\"schema_providers\")));",
```

Comment:
This looks good -- took me a bit to understand where this is accessed from the Postman UI (for example right click providers/edit/test)